README.md

```diff
@@ -42,7 +42,7 @@ One problem with this approach is that it changes the meaning of
 avoid defining the following identifiers:
 
  1. `Lambda` and `λ`
- 2. `?`, `+?`, and `-?`
+ 2. `*`, `+*`, and `-*`
```

Comment:
```suggestion
 2. `*`, `+*`, and `-*`
 2. `?`, `+?`, and `-?` ([deprecated syntax](https://github.com/typelevel/kind-projector/issues/108); same meaning as the above)
```

Reply:
👍

Reply:
I've merged this including your suggestion, @Daenyth.